.buildkite/pipeline.build_core.yml

```diff
@@ -1,31 +1,19 @@
 #ci:group=:book: core tests (civ2)
 
-- label: ":python: small & medium"
-  conditions: ["RAY_CI_PYTHON_AFFECTED"]
-  instance_size: medium
-  parallelism: 3
-  commands:
-    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
-    - DL=1 ./ci/env/install-dependencies.sh
-    - ./ci/env/env_info.sh
-    - bazel run //ci/ray_ci -- //python/ray/tests/... //python/ray/dag/... core --size small,medium --concurrency "$${BUILDKITE_PARALLEL_JOB_COUNT}" --shard "$${BUILDKITE_PARALLEL_JOB}"
-
-- label: ":python: large"
+- label: ":python:"
   conditions: ["RAY_CI_PYTHON_AFFECTED"]
   instance_size: large
-  parallelism: 3
+  parallelism: 6
```

Comment:
why do we need to double the shards don buildkite jobs?

Reply:
oh, it is because this is small+medium+large now?

Reply:
that's right, size is now all together